qctrlopencontrols/driven_controls/predefined.py

```diff
@@ -217,25 +214,34 @@ def _new_primitive_control(
     maximum_rabi_rate: float = 2.0 * np.pi,
```

Comment:
Doesn't seem to be a clear way to do this, since Sphinx doesn't even see the fact that it's an `np.pi` (it gets expanded to its float value as soon as Python loads the module). See https://stackoverflow.com/questions/29257961/sphinx-documentation-how-to-disable-default-argument-expansion too.